README.md

```diff
@@ -84,6 +84,7 @@ Structure of the `config.json` file is as follows:
 		"defaultBuildCommand": "docker build --tag {{.IMAGE_NAME}}:{{.IMAGE_VERSION}} --tag {{.DEFAULT_PUSH_REGISTRY}}/{{.IMAGE_NAME}}:{{.IMAGE_VERSION}} --tag {{.DEFAULT_PULL_REGISTRY}}/{{.IMAGE_NAME}}:{{.IMAGE_VERSION}} {{.DOCKERFILE_DIR}}",
 		"defaultPushCommand": "docker push {{.DEFAULT_PUSH_REGISTRY}}/{{.IMAGE_NAME}}:{{.IMAGE_VERSION}}"
 	},
+	"reportFilename": "custom-report-filename.json",
```

Comment:
Please descibe in a readme what this property can be used for. Also can this be a full path or just a name? If a full path, consider renaming to `reportFilePath`

Reply:
doc added. name is done in the exact same convention as dump-latest-versions' file-name. just a name, not a path.